### DISCUSSION TITLE
Keyboard shortcut to toggle notes search

### DISCUSSION THREAD

**Discussion by sailr7** [standardnotes/forum#3390] [Suggestions]
Is there a keyboard shortcut to toggle notes search? 

I am not referring to searching within an active note, but doing a global search of all notes. This would be quite useful to have.

**Comment by effieeee** [comment]
Hi @ztwhaley, as far as I'm aware, there isn't a dedicated shortcut for the global search at the moment, though I'll add this as suggestion 🙏

**Reply by sailr7** [reply]
Thanks for the feedback. In the meantime, do you think this functionality is something that I could easily have built as a plug-in?

**Reply by effieeee** [reply]
I wouldn't be able to say for sure, but we do have some guides on how to get started with building plugins on our help page!
https://standardnotes.com/help/plugins/intro

**Reply by munkmode** [reply]
+1, I desperately want this shortcut also.
